stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java

```diff
@@ -72,6 +72,11 @@ public static void main(String[] args) {
     mainInternal(args, new StressMasterBench());
   }
 
+  @Override
+  public String getBenchDescription() {
+    return "";
```

Comment:
Why are these empty string?

Reply:
cuz it's not the test added by shawn, shawn has no knowledge on those tests, need you guys to add the description later